docs/porting/libraries.md

```diff
@@ -1,5 +1,304 @@
+   
+5. Repeat this process as needed for different projects.
+6. Once you have a plan, execute that plan.
+ 
+The analysis phase could take some time depending on how large your codebase is.  Spending time in this phase to thoroughly understand the scope of changes needed and to develop a plan can save you a lot of time in the long run, particularly if you have a more complex codebase.
+
+Your plan could involve making significant changes to your codebase while still targeting .NET Framework 4.6.1, making this a more structured version of the previous approach.  How you go about executing your plan will be dependent on your codebase.
+
+### Mixing approaches
+
+It's likely that you'll mix the above approaches on a per-project basis.  You should do what makes the most sense to you and for your codebase.
+
+## Picking your project system in Visual Studio: xproj or traditional projects
+
+In addition to a new project model, .NET Core introduces a new project system, xproj, in Visual Studio.  When porting your code in Visual Studio, you'll have to pick the correct project system based on your needs.
+
+### When to pick xproj
+
+The new Xproj project system in Visual Studio utilizes the capabilities of the `project.json`-based project model to offer two major features over existing project types: seamless multitargeting by building multiple assemblies and the ability to directly generate a NuGet package on build.
+
+However, it comes at the cost of lacking certain features you may use, such as:
+
+- Support for F# or VB
+- Generating satellite assemblies with localized resource strings
+- Directly referencing a `.dll` file on the filesystem
+
+If your project needs are relatively minimal and you can take advantage of the new features of xproj, you should pick it as your project system.  This can be done in Visual Studio as such:
+
+1. Ensure you are using Visual Studio 2015 or later.
+2. File | New Project.
+3. Select ".NET Core" under Visual C#.
+4. Selecting the "Class Library (.NET Core)" template. 
+
+### When to pick a traditional project system
+
+You can target .NET Core with the traditional project system in Visual Studio, by creating a Portable Class Library (PCL) and selecting ".NET Core" in the project configuration dialog.
+
+If you have more intricate project system needs, this should be your choice.  It may also be the best choice if you're mixing .NET Core and .NET Framework code in a larger solution.  Note that if you wish to multitarget by generating platform-specific assemblies like with the `xproj` project system, you'll need to create a "Bait and Switch" PCL, [as described here](https://blogs.msdn.microsoft.com/dsplaisted/2012/08/27/how-to-make-portable-class-libraries-work-for-you/).
```

Comment:
Links should have a name for SEO purposes and not use here, etc. Suggested rewriting:
you'll need to create a "Bait and Switch" PCL. For more information, see [How to Make Portable Class Libraries Work for You](https://blogs.msdn.microsoft.com/dsplaisted/2012/08/27/how-to-make-portable-class-libraries-work-for-you/).